### PR TITLE
Refund amount

### PR DESCRIPTION
**Pull request by asogwachiuba**
What is in the PR:
Amount to be refunded was added on the refund screen.

Simulator Screen Shot - iPhone SE (3rd generation) - 2023-06-17 at 19 12 38

### CODE REVIEW

**Comment by divincoder** [comment]
why is the amount in dollars

**Comment by asogwachiuba** [comment]
> why is the amount in dollars

Ahhhhhh the design was in dollars. Didn't even occur to me. I will make the changes today.

File: lib/features/refunds/refund_home_view.dart [hunk 1]
```diff
@@ -237,7 +249,31 @@ class _RefundHomeViewState extends State<RefundHomeView> {
...
+                      TextSpan(
+                        text: 'Amount to be refunded  ',
+                        style: TextStyle(
+                          color: Color(0xFF3F3F3F),
+                          fontSize: 15,
+                          fontWeight: FontWeight.w400,
+                        ),
+                      ),
+                      TextSpan(
+                          text: '\₦ ${getRefundAmount()}',
```
Comment (divincoder) [inline lib/features/refunds/refund_home_view.dart:267]: There is a formatter for amount. Use it here.

Comment (asogwachiuba) [reply lib/features/refunds/refund_home_view.dart:267]: Ohh okay.

Comment (asogwachiuba) [reply lib/features/refunds/refund_home_view.dart:267]: Done

Comment (divincoder) [reply lib/features/refunds/refund_home_view.dart:267]: Did you confirm the formatter doesn't return a naira sign as well?

**Review by divincoder — APPROVED** [review]

**Pull request merged**